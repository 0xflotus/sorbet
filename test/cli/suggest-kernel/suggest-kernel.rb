# typed: true
module M
  def foo
    raise "hi"
  end
end
