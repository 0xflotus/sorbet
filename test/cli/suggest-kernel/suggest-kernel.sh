#!/bin/sh
#!/bin/bash
main/sorbet test/cli/suggest-kernel/suggest-kernel.rb 2>&1
