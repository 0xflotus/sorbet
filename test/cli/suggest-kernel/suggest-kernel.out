test/cli/suggest-kernel/suggest-kernel.rb:4: Method `raise` does not exist on `Opus::DB::Model` http://go/e/7003
     4 |    raise "hi"
            ^^^^^^^^^^
  Did you mean to `include Kernel` in this module?
Errors: 1
